src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -144,7 +144,7 @@ public class AgentKnobs
             nameof(HttpTrace),
             "Enable http trace if true",
             new EnvironmentKnobSource("VSTS_AGENT_HTTPTRACE"),
-            new BuiltInDefaultKnobSource("false"));
+            new BuiltInDefaultKnobSource("true"));
```

Comment:
@alex-peck did we mean to change this value?

Reply:
Crap! Didn't mean to check this in